### ISSUE TITLE
Add Google Analytics page views count to WP Admin's Pages list view. 

### ISSUE THREAD

**Issue by jimmymadon** [google/site-kit-wp#6503]
## Feature Description

This issue was "split out" from #5418 to implement the new GA Page Views column for the Pages list view. 

--------------

Considering adding a total Google Analytics view count on both the pages and posts tab, as per the screenshot below, once Site Kit is set up with Analytics connected. This could be a sortable column, as some of the other columns are within the same list views. 

![image](https://user-images.githubusercontent.com/41326532/175268060-bf542971-4662-469c-aaee-33797c2ecd10.png)

This is a feature request on behalf of [one user in the support forums](https://wordpress.org/support/topic/a-sortable-view-menu/#post-15761795). 

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Given GA4 is connected, when an admin visits the Pages list view, then they should see a new column as follows:
    * Column title: __GA Page Views__ (translatable string)
    * Column data: Metric should be `screenPageViews` and dimension should be `pagePathPlusQueryString` for the Page's URL. (Use existing GA4 datastore infrastructure to fetch this data).
    * The column should appear after the "Author" column (if it exists), else it should appear before the "Date" column.
    * Sortable: Sort based on the number of `screenPageViews`.
* The above will require Site Kit's assets to be loaded on the `Pages` wp-admin page similar to how we load them for the WP Dashboard.

## Implementation Brief

Note to IB reviewer: Plenty of [docs](https://developer.wordpress.org/reference/hooks/manage_post_type_posts_columns/) and [guides](https://awhitepixel.com/blog/modify-add-custom-columns-post-list-wordpress-admin/) on how to implement the above. 

- TBC: Waiting for an IB review of #5418 to make a final decision on the approach here.
- In `includes/Modules/Analytics_4/Page_Views_Post_List_Column.php` (being added in #5418) file:
    - Update the `register()` method:
	    - Besides adding the `manage_columns` and `manage_custom_column` as callback functions to the `manage_post_posts_columns` and `manage_post_posts_custom_column` hooks, they should also be added to the `manage_page_posts_columns` and `manage_page_posts_custom_column` hooks respectively.

### Test Coverage

- No tests need to be added/updated.

## QA Brief

* 

## Changelog entry

*

**Comment by tofumatt** [comment]
Similar question here as I had for the "Posts" view here: https://github.com/google/site-kit-wp/issues/5418#issuecomment-1412847949

Is _"The above will require Site Kit's assets to be loaded on the Pages wp-admin page similar to how we load them for the WP Dashboard."_ needed as part of the ACs here?

**Comment by tofumatt** [comment]
Looks good, moving to IB 👍🏻

**Comment by sashadoes** [comment]
Unassigning myself for now due to ongoing conversations at #5418 regarding filtering eg: athttps://github.com/google/site-kit-wp/issues/5418#issuecomment-1420564334

**Comment by bethanylang** [comment]
@eugene-manuilov Are you planning to work on this soon? If not, can you please unassign yourself so someone else can pick up? Thank you!

**Comment by eugene-manuilov** [comment]
Same as in the similar task #5418, going to discuss it next week on our engineering round table call.

**Comment by benbowler** [comment]
@eugene-manuilov did a decision get made on this issue in the round table? I can't see references to it in Slack. Happy to pick it up.

**Comment by eugene-manuilov** [comment]
No, we haven't managed to discuss this yet. I'll update this ticket once we do it.

**Comment by aaemnnosttv** [comment]
Closing this in favor of the original since it doesn't require a post type specific implementation.
